cinema/views.py

```diff
@@ -21,24 +32,49 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
```

Comment:
It is a better approach to place mixins on a first position in inheritance here and in the other classes as well